planning/mission_planner/src/mission_planner/mission_planner.cpp

```diff
@@ -367,6 +488,11 @@ void MissionPlanner::on_modified_goal(const ModifiedGoal::Message::ConstSharedPt
     RCLCPP_ERROR(get_logger(), "Normal route has not set yet.");
     return;
   }
+  if (mrm_route_) {
```

Comment:
Isn't modified_goal necessary even in MRM? There may be obstacles at the goal of the mrm_route.

Reply:
I'll submit a new PR for handling modified goal during the MRM state next week, I don't want to change goals in the emergency state

Reply:
I see.